rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -29,219 +29,441 @@
+    return (brightness_image)
 
-    2 - increase brightness twice
 
-    bytes_per_sample_hint (int, optional, default = 0) – Output size hint (bytes), per sample. The memory will be preallocated if it uses GPU or page-locked memory
+def brightness_fixed(*inputs, alpha=1.0, beta=0.0,
+                     rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
+    """
+    inputs - the input image passed to the augmentation
+    
+    alpha (float, optional, default = 1.0) - brightness multiplier. Values >= 0 are accepted. For example: 0 - black image, 1 - no change, 2 - increase brightness twice
 
-    image_type (int, optional, default = 0) – The color space of input and output image
+    beta (float, optional, default = 0.0) - brightness shift
 
-    preserve (bool, optional, default = False) – Do not remove the Op from the graph even if its outputs are unused.
+    rocal_tensor_output_layout (int, optional, default = types.NHWC) - tensor layout for the augmentation output
 
-    seed (int, optional, default = -1) – Random seed (If not provided it will be populated based on the global seed of the pipeline)
+    rocal_tensor_output_datatype (int, optional, default = types.UINT8) - tensor dtype for the augmentation output
     """
-
-    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "alpha": None, "beta": None}
-    brightness_image = b.Brightness(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    # pybind call arguments
+    kwargs_pybind = {"input_image": inputs[0], "is_output": False, "alpha": alpha, "beta": beta,
+                     "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+    brightness_image = b.BrightnessFixed(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (brightness_image)
 
-def brightness_fixed(*inputs, alpha=None, beta=None, seed=-1, device=None):
-    alpha = b.CreateFloatParameter(alpha) if isinstance(alpha, float) else alpha
-    beta = b.CreateFloatParameter(beta) if isinstance(beta, float) else beta
-    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "alpha": alpha, "beta": beta}
-    brightness_image = b.Brightness(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
-    return (brightness_image)
 
-def lens_correction(*inputs, strength =None, zoom = None):
-    strength = b.CreateFloatParameter(strength) if isinstance(
-        strength, float) else strength
-    zoom = b.CreateFloatParameter(zoom) if isinstance(zoom, float) else zoom
-    kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "strength": strength, "zoom": zoom}
-    len_corrected_image = b.LensCorrection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+def lens_correction(*inputs, strength=None, zoom=None,
+                    rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
```

Comment:
Move this to prev line if the length of all the arguments is less. Check in the entire file once